packages/config/src/config.js

```diff
@@ -0,0 +1,47 @@
+import { pickBy } from 'ramda'
+import deepMerge from './utils/merge'
+
+let layers = []
+
+let activeConfiguration = {};
+
+function define(configuration) {
+  layers.push(configuration)
+}
+
+function init(profiles) {
+
+  // reverse the order of the profiles, least important handled first
+  // and overridden with values from the more important profiles
+  if (profiles) profiles.reverse()
+
```

Comment:
The bigger problem is that a mutable change is applied to the argument of the fn which is a *bad thing*. Please use a `ramda` variant of reverse as it will return new version of the array.